utils/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+# Build the compiler without GWT.
+mvn -DskipTests -pl externs/pom.xml,pom-main.xml,pom-main-shaded.xml
```

Comment:
Handy for debugging local build.

Reply:
Can you move the utils/ scripts to be inside src/com/google/javascript/jscomp/?

Reply:
Sure - there's just an existing `/travis_utils` dir, and I am going to be adding a `runtime_tests.sh` script for jscomp runtime tests as part of a separate PR, and I figured this would be a good time to raise the idea of just moving those all to a root-level `utils/` dir for organization.

Will move them, just something to think about.